OWNERS

```diff
@@ -4,10 +4,10 @@ approvers:
   - xmudrii
   - kron4eg
   - xrstf
-  - eqrx
+  - moelsayed
```

Comment:
Nit: can we sort this file alphabetically?

Reply:
done